package.json

```diff
@@ -7,7 +7,7 @@
     "preversion": "npm test",
     "postversion": "npm run build-docs && git push && git push --tags",
     "test": "NODE_ENV=test mocha --compilers js:babel-core/register -R spec src/test/*.js src/modules/**/test/*.js",
-    "compile": "babel src --out-dir lib --ignore src/modules/web-theme-default/assets; cp -r src/modules/web-theme-default/assets lib/modules/web-theme-default/; cp -r src/templates lib/templates; cp -r src/modules/web-theme-default/partials lib/modules/web-theme-default/; cp -r src/modules/web-theme-default/layouts lib/modules/web-theme-default/; cp -r src/modules/web-nav/templates lib/modules/web-nav/",
+    "compile": "babel src --out-dir lib --ignore src/modules/web-theme-default/assets; cp -r src/modules/web-theme-default/assets lib/modules/web-theme-default/; cp -r src/templates lib/; cp -r src/modules/web-theme-default/partials lib/modules/web-theme-default/; cp -r src/modules/web-theme-default/layouts lib/modules/web-theme-default/; cp -r src/modules/web-nav/templates lib/modules/web-nav/; cp -r src/modules/web-actions/templates lib/modules/web-actions/",
```

Comment:
Not sure if you're more comfortable with a shell loop in here (that would not need to be explicit for each new sub-module) or with going back to a root templates dir that repeats the sub-module structure. Former might look like 
```
for $each in `ls src/modules/`; do if [ -d src/modules/$each/templates ]; then; cp -r src/modules/$each/templates lib/modules/$each; fi; done;
```